### PR TITLE
Ql 6 build checkbox

### PR DESCRIPTION
**Pull request by peterianmuller**

### CODE REVIEW

File: src/components/Checkbox/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// display error and correct states 
+
+export const CheckBox = ({ onClick, id, name, value, error, success, label, ...rest }) => {
+  return (
+    <label>
+      <input type="checkbox" id={id} name={name} value={value} onClick={onClick} error={error} success={success} label={label}></input>
```
Comment (svey) [inline src/components/Checkbox/index.js:9]:
```suggestion
      <input type="checkbox" id={id} name={name} value={value} onClick={onClick} error={error} success={success} label={label} />
```

Comment (svey) [inline src/components/Checkbox/index.js:9]: where do these error and success go?

Comment (svey) [inline src/components/Checkbox/index.js:9]:
```suggestion
      // @todo error and success states still need to be handled
      <input type="checkbox" id={id} name={name} value={value} onClick={onClick} error={error} success={success} label={label}></input>
```

Comment (peterianmuller) [reply src/components/Checkbox/index.js:9]: Removed closing tag from input b/c no children

File: src/components/Checkbox/index.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// display error and correct states 
+
+export const CheckBox = ({ onClick, id, name, value, error, success, label, ...rest }) => {
+  return (
+    <label>
+      <input type="checkbox" id={id} name={name} value={value} onClick={onClick} error={error} success={success} label={label} />
```
Comment (peterianmuller) [inline src/components/Checkbox/index.js:9]:
```suggestion
      {/*@todo error and success states still need to be handled */}
      <input type="checkbox" id={id} name={name} value={value} onClick={onClick} error={error} success={success} label={label} />
```

**Pull request merged**